src/material-examples/table-pagination/table-pagination-example.html

```diff
@@ -0,0 +1,41 @@
+<div class="example-container mat-elevation-z8">
+  <div class="example-header"> Users </div>
+
+  <cdk-table #table [dataSource]="dataSource">
+
```

Comment:
Maybe add a comment here like

"Note that these columns can be defined in any order. The actual rendered columns are set as a property on the row definition"